Documentation

kristindew opened the pull request:
To add documentation for db models to the master.

AllieDeford committed:
Merge pull request #7 from SCIP-app/master

Update Documentation Branch to Current Master

kristindew committed:
Added documentation for db models with a few questions marked for tomorrow. Also updated the LoginActivity populate database function with a long value instead of a date string for MemsCap object b/c it was angry.

AllieDeford commented:
Let's go over some of these questions in the pair programming session tomorrow! But I think you've made a good start here

kristindew committed:
Corrected documentation for db models and did documentation for DatabaseHelper object

kristindew committed:
Added Google Analytics default tracking

Compiles and but am crashing from a runtime error due to a null pointer
exception involving 'long
scip.app.models.Participant.getParticipantId()'

AllieDeford committed:
merged with current master

kristindew committed:
Some of the documentation
